lib/src/main/res/drawable/ic_address_book.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="24dp" android:viewportHeight="1000"
```

Comment:
If we can try to remove this icon and use an `Icons.Default`, I think it would be better